source/magnifier/commands.py

```diff
@@ -0,0 +1,213 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2025 NV Access Limited, Antoine Haffreingue
+# This file may be used under the terms of the GNU General Public License, version 2 or later, as modified by the NVDA license.
+# For full terms and any additional permissions, see the NVDA license file: https://github.com/nvaccess/nvda/blob/master/copying.txt
+
+"""
+Keyboard commands for the magnifier module
+Contains the command functions and their logic for keyboard shortcuts
+"""
+
+import ui
+from . import getMagnifier, initialize, terminate
+from .config import (
+	getCurrentZoomLevel,
+	setDefaultZoomLevel,
+	getCurrentFilter,
+	setDefaultFilter,
+	getCurrentFullscreenMode,
+	setDefaultFullscreenMode,
+	shouldSaveShortcutChanges,
+)
+from .magnifier import Magnifier
+from .fullscreenMagnifier import FullScreenMagnifier
+from .utils.types import (
+	Filter,
+	Direction,
+	MagnifierType,
+	FullScreenMode,
+	MagnifierAction,
+)
+from logHandler import log
+
+
+def toggleMagnifier():
+	"""Toggle the NVDA magnifier on/off."""
+	magnifier: Magnifier = getMagnifier()
+	if magnifier and magnifier.isActive:
+		# Stop magnifier
+		terminate()
+		ui.message(
+			_(
```

Comment:
Please use pgettext for new strings added to this _magnifier submodule

```suggestion
			pgettext(
				"magnifier",
```